privacyidea/static/components/token/views/token.details.html

```diff
@@ -515,7 +515,7 @@ <h4 ng-show="token.user_id" translate>Assigned User</h4>
                     {{ token.username }}
                 </a></td>
                 <td>
-                    <button ng-hide="token.locked"
+                    <button ng-hide="token.locked || !checkRight('unassign')"
```

Comment:
We are using the webui policy `hide_buttons` to, well, hide buttons...
Could You change this to:
```
      <button ng-hide="token.locked || !checkRight('unassign') && hide_buttons"
              ng-disabled="!checkRight('unassign')"
              ...
```
That way we can disable and (if the policy is set) hide the button.

Reply:
OK, the changes are commited